platform/linuxbsd/detect.py

```diff
@@ -339,6 +339,11 @@ def configure(env):
     else:
         print("ALSA libraries not found, disabling driver")
 
+    if os.system("pkg-config --exists sndio") == 0:  # 0 means found
+        print("Enabling sndio")
+        env.Append(CPPDEFINES=["SNDIO_ENABLED"])
+        env.ParseConfig("pkg-config --cflags --libs sndio")
+
```

Comment:
It's an optional driver, probably it should have build flag to enable/disable it.

Similar to the flag used for pulse audio:
https://github.com/godotengine/godot/blob/master/platform/linuxbsd/detect.py#L74
https://github.com/godotengine/godot/blob/master/platform/linuxbsd/detect.py#L343

Reply:
sure, added :)